noaa/track.json

```diff
@@ -0,0 +1,28 @@
+{% import "rally.helpers" as rally with context %}
+
+{
+  "short-description": "Daily weather measurement summaries from around the globe.",
+  "description": "Indexes 10M+ weather measurement summaries from NOAA.",
```

Comment:
@danielmitterdorfer The doc count is actually 10914068, so I'll just update it to that. I would expect Rally to fail with an error, because the `document-count` in `track.json` was incorrect.